tests/test_ic8report.py

```diff
@@ -0,0 +1,55 @@
+import os
+from typing import Dict
+
+import pandas as pd
+import pathlib
+
+path_to_data_for_tests = (
+    pathlib.Path(os.path.dirname(__file__)) / "code_and_data_for_tests"
+)
+
+def run_ic8_report(tmpdir: pathlib.Path) -> Dict:
+    """Returns the results generated by the running the Report class. (Also generates that Excel file in a temporary
+    location.)"""
+
+    from scripts.ic8.analyses.main_results_for_investment_case import get_report
+
+    r = get_report(
+        load_data_from_raw_files=True,
+        do_checks=False,
+        run_analysis=True,
+    )
+
+    # Generate report (checking that it can be written to an Excel file)
+    filename = tmpdir / 'test_report_ic8.xlsx'
+    rtn_from_return = r.report(filename)
+
+    # Return the results of the report
+    return rtn_from_return
+
+"""
+Warning to ensure this runs make sure in the parameters.toml you choose the full list of countries for portfolio and 
+modelled countries, otherwise this code will throw and error. 
+"""
```

Comment:
rather than a warning, better to force this to be the case by over-writing parameters in this test, to ensure this.

(Or putting in a switch that enables this to happen more seamlessly - though this isnt' necessary)

Reply:
@tbhallett  I was thinking the same. At the moment I am trying to make it work (test is still failing) and then I will have a look if I can make a switch for this

Reply:
Btw tests passed for me